shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -731,3 +735,39 @@ unitName
 procedureName
     : identifier
     ;
+
+validTimeColumn
+    : columnName
+    ;
+
+attrDim
+    : identifier
+    ;
+
+hierarchyName
+    : (owner DOT_)? name
+    ;
+
+analyticViewName
+    : (owner DOT_)? name
+    ;
+
+samplePercent
+    : numberLiterals
+    ;
+
+seedValue
+    : numberLiterals
+    ;
+
+condition
+    : expr
+    ;
+
+variableName
+    : identifier | STRING_
+    ;
+
+externalTableDataProps
```

Comment:
It's better to move the rule to DDLStatement. And rename the rule and copy the rule in DMLStatement.